### PR TITLE
Financial Data

### PR DESCRIPTION
**Pull request by eanders**

### CODE REVIEW

File: drivers/financial/app/models/financial/client_import.rb [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    end
+
+    def import!(force = false)
+      return unless check_hour || force
+
+      start_import
+      fetch_and_load
+      complete_import
+      post_process
+    end
```
Comment (awisema) [inline drivers/financial/app/models/financial/client_import.rb:21]: This replaces `import!` in `CsvImportConcern` -- should this be moved to the concern?

File: drivers/financial/spec/financial_spec.rb [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# GrdaWarehouse::CustomImports::Config.create(s3_prefix: "#{Rails.env}/#{Financial::Import.import_prefix}/", data_source_id: 2, s3_region: 'us-east-1', s3_bucket: 'financial-bucket', active: true, import_type: 'Financial::Import', user: User.system_user, import_hour: 6, s3_access_key_id: 'local_access_key', s3_secret_access_key: 'local_secret_key')
```
Comment (awisema) [inline drivers/financial/spec/financial_spec.rb]: Are we missing a commit here?

**Review by awisema — CHANGES_REQUESTED** [review]

File: drivers/financial/spec/financial_spec.rb (hunk 2 above)
Comment (eanders) [reply drivers/financial/spec/financial_spec.rb]: oh, no.  I meant to add some tests, but ended up just testing things manually.  I'll add a TODO for future development.

File: drivers/financial/app/models/financial/client_import.rb (hunk 1 above)
Comment (eanders) [reply drivers/financial/app/models/financial/client_import.rb:21]: No, this is an intentional override, it differs from the other two importers that don't have post_process.
The other alternative was to make an empty post_process method and override it.... Same difference as far as I was concerned.

**Review by awisema — APPROVED** [review]

**Pull request merged**